### ISSUE TITLE
Cannot install library through npm

### ISSUE THREAD

**Issue by lucasmotta** [valentinpalkovic/prisma-json-schema-generator#6]
Hey there, first thanks for creating this package! I've seen your comment on the Prisma repo that you created your own generator to create a json file, which is exactly what I'm after.

The problem is that I cannot install the library through npm:

```
npm install prisma-json-schema-generator --save-dev
```

```
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /Users/xxxxx/xxxxx/node_modules/prisma-json-schema-generator/dist/cli.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/xxxxx/xxxxx/node_modules/prisma-json-schema-generator/dist/cli.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

My environment:

```
macOS: 10.15.6
node: 12.18.4
npm: 6.14.6
```

**Comment by valentinpalkovic** [comment]
@lucasmotta Can you provide some additional information for reproduction reasons? 

- os
- node version
- npm Version

**Comment by lucasmotta** [comment]
@valentinpalkovic I just updated the main description with my environment - I forgot to add it originally.

**Comment by valentinpalkovic** [comment]
@lucasmotta Please try it with the newest released version (1.1.3). Indeed, there was a wrong configuration in the `package.json`. Thanks for reporting this bug!

**Comment by lucasmotta** [comment]
Nice one @valentinpalkovic, now it works fine! Thank you 👍
